### ISSUE TITLE
SSL Problems during installation

### ISSUE THREAD

**Issue by ManolisNikolakakis** [CERN/TIGRE#564]
## Expected Behavior
I expected installation of tigre from pip, but my machine successfully installed other packages. installation has failed in two different machines, a MacOS and an Ubuntu

## Actual Behavior

File "/Users/enikolak/anaconda3/lib/python3.9/urllib/request.py", line 641, in http_error_default
      raise HTTPError(req.full_url, code, msg, hdrs, fp)
urllib.error.HTTPError: HTTP Error 403: SSL is required
[end of output]

## Code to reproduce the problem (If applicable)
pip install tigre

```

```

## Specifications 

  - MATLAB/python version: 3.8.10
  - OS: 20.04.1-Ubuntu SMP
  - CUDA version: Cuda compilation tools, release 10.1, V10.1.243

**Comment by AnderBiguri** [comment]
This has nothing to do with TIGRE and instead with something about your installation. Likely the machine does not have internet somehow and this you can't install the required packages, e.g. numpy. I suspect `conda install numpy` or `pop install numpy` causes the same error.

**Comment by ManolisNikolakakis** [comment]
Hello and thank you for your response. 

I mentioned in the original post that I've been able to install other packages such as numpy and other packages and I did this test in two different machines. It's only TIGRE that was throwing this error.

However I agree that this doesn't seem like a TIGRE issue, I was hoping maybe someone else has also experienced this issue. Might be a PyPI mirror issue.

I'll close this ticket, thanks for your advice.
